packages/react-core/src/components/LoginPage/examples/LoginPageLanguageSelect.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import brandImg2 from './brandImgColor2.svg';
+import {
+  LoginFooterItem,
+  LoginForm,
+  LoginMainFooterBandItem,
+  LoginMainFooterLinksItem,
+  LoginPage,
+  ListItem,
+  ListVariant,
+  Select,
+  SelectOption
+} from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+
+export const LoginPageLanguageSelect: React.FunctionComponent = () => {
+  const [showHelperText, setShowHelperText] = React.useState<boolean>(false);
+  const [username, setUsername] = React.useState<string>('');
+  const [isValidUsername, setIsValidUsername] = React.useState<boolean>(true);
+  const [password, setPassword] = React.useState<string>('');
+  const [isValidPassword, setIsValidPassword] = React.useState<boolean>(true);
+  const [isRememberMeChecked, setIsRememberMeChecked] = React.useState<boolean>(false);
+  const [isHeaderUtilsOpen, setIsHeaderUtilsOpen] = React.useState<boolean>(false);
+  const [selectedHeaderUtils, setSelectedHeaderUtils] = React.useState<string | SelectOption>();
+
+  const headerUtilsOptions = [
+    <SelectOption key={0} value="English" />,
```

Comment:
Can this be selected in the menu, too? FWIW I found this [filterable table demo](https://www.patternfly.org/v4/components/table/react-demos#filterable) that has "status" as initially selected, and if you open the menu, it's selected in the menu, too.

Reply:
got it!